crate_universe/src/config.rs

```diff
@@ -181,6 +186,43 @@ pub enum Checksumish {
     },
 }
 
+#[derive(Debug, Default, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize, Clone)]
+pub enum AliasRule {
+    #[default]
+    #[serde(rename = "default")]
```

Comment:
I would be tempted to call this `alias` or `native.alias` rather than `default`? WDYT?

Reply:
Will do!